flow/network.h

```diff
@@ -31,55 +31,65 @@
 #include "flow/IRandom.h"
 #include "fdbrpc/crc32c.h"
 
-enum {
-	TaskMaxPriority = 1000000,
-	TaskRunCycleFunction = 20000,
-	TaskFlushTrace = 10500,
-	TaskWriteSocket = 10000,
-	TaskPollEIO = 9900,
-	TaskDiskIOComplete = 9150,
-	TaskLoadBalancedEndpoint = 9000,
-	TaskReadSocket = 9000,
-	TaskCoordinationReply = 8810,
-	TaskCoordination = 8800,
-	TaskFailureMonitor = 8700,
-	TaskResolutionMetrics = 8700,
-	TaskClusterController = 8650,
-	TaskProxyStorageRejoin = 8645,
-	TaskProxyCommitDispatcher = 8640,
-	TaskTLogQueuingMetrics = 8620,
-	TaskTLogPop = 8610,
-	TaskTLogPeekReply = 8600,
-	TaskTLogPeek = 8590,
-	TaskTLogCommitReply = 8580,
-	TaskTLogCommit = 8570,
-	TaskProxyGetRawCommittedVersion = 8565,
-	TaskProxyResolverReply = 8560,
-	TaskProxyCommitBatcher = 8550,
-	TaskProxyCommit = 8540,
-	TaskTLogConfirmRunningReply = 8530,
-	TaskTLogConfirmRunning = 8520,
-	TaskProxyGRVTimer = 8510,
-	TaskProxyGetConsistentReadVersion = 8500,
-	TaskDefaultPromiseEndpoint = 8000,
-	TaskDefaultOnMainThread = 7500,
-	TaskDefaultDelay = 7010,
-	TaskDefaultYield = 7000,
-	TaskDiskWrite = 5030,
-	TaskStorage = 5020,
-	TaskDiskRead = 5010,
-	TaskDefaultEndpoint = 5000,
-	TaskUnknownEndpoint = 4000,
-	TaskMoveKeys = 3550,
-	TaskDataDistributionLaunch = 3530,
-	TaskRatekeeper = 3510,
-	TaskDataDistribution = 3500,
-	TaskUpdateStorage = 3000,
-	TaskTLogSpilledPeekReply = 2800,
-	TaskLowPriority = 2000,
-	TaskMinPriority = 1000
+enum class TaskPriority {
+	Max = 1000000,
+	RunCycleFunction = 20000,
+	FlushTrace = 10500,
+	WriteSocket = 10000,
+	PollEIO = 9900,
+	DiskIOComplete = 9150,
+	LoadBalancedEndpoint = 9000,
+	ReadSocket = 9000,
+	CoordinationReply = 8810,
+	Coordination = 8800,
+	FailureMonitor = 8700,
+	ResolutionMetrics = 8700,
+	ClusterController = 8650,
+	ProxyStorageRejoin = 8645,
+	ProxyCommitDispatcher = 8640,
+	TLogQueuingMetrics = 8620,
+	TLogPop = 8610,
+	TLogPeekReply = 8600,
+	TLogPeek = 8590,
+	TLogCommitReply = 8580,
+	TLogCommit = 8570,
+	ProxyGetRawCommittedVersion = 8565,
+	ProxyResolverReply = 8560,
+	ProxyCommitBatcher = 8550,
+	ProxyCommit = 8540,
+	TLogConfirmRunningReply = 8530,
+	TLogConfirmRunning = 8520,
+	ProxyGRVTimer = 8510,
+	ProxyGetConsistentReadVersion = 8500,
+	DefaultPromiseEndpoint = 8000,
+	DefaultOnMainThread = 7500,
+	DefaultDelay = 7010,
+	DefaultYield = 7000,
+	DiskWrite = 5030,
+	Storage = 5020,
+	DiskRead = 5010,
+	DefaultEndpoint = 5000,
+	UnknownEndpoint = 4000,
+	MoveKeys = 3550,
+	DataDistributionLaunch = 3530,
+	Ratekeeper = 3510,
+	DataDistribution = 3500,
+	UpdateStorage = 3000,
+	TLogSpilledPeekReply = 2800,
+	Low = 2000,
+
+	Min = 1000,
+	Zero = 0
 };
 
+inline TaskPriority incrementPriority(TaskPriority p) {
+	return static_cast<TaskPriority>( static_cast<uint64_t>(p) + 1 );
+}
+
+inline TaskPriority decrementPriority(TaskPriority p) {
+	return static_cast<TaskPriority>( static_cast<uint64_t>(p) + 1 );
```

Comment:
I also fixed the + being a + 😞

Reply:
Oops, I missed that too, glad you saw it.